nservicebus/upgrades/testing-5to6.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Upgrade NServiceBus.Testing Version 5 to 6
+summary: Instructions on how to upgrade NServiceBus.Testing Version 5 to 6.
+tags:
+ - upgrade
+ - migration
+related:
+- nservicebus/testing
+---
+
+
+## Move to .NET 4.5.2
```

Comment:
i think just start with "an upgrade to v6 core will also be required" and link. no need to duplicate the .net 4.5.2 content here